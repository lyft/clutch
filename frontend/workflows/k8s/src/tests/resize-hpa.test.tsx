import React from "react";
import { shallow } from "enzyme";

import ResizeHPA from "../resize-hpa";

describe("Resize Autoscaling Group workflow", () => {
  let component;

  beforeAll(() => {
    component = shallow(<ResizeHPA resolverType="clutch.aws.k8s.v1.HPA" />);
  });

  it("renders correctly", () => {
    expect(component.debug()).toMatchSnapshot();
  });
});
