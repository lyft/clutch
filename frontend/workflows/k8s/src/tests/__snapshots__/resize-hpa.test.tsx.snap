// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resize Autoscaling Group workflow renders correctly 1`] = `
"<Wizard dataLayout={{...}} heading={[undefined]}>
  <HPAIdentifier name=\\"Lookup\\" resolverType=\\"clutch.aws.k8s.v1.HPA\\" />
  <HPADetails name=\\"Modify\\" />
  <Confirm name=\\"Confirmation\\" notes={{...}} />
</Wizard>"
`;
